.github/workflows/build_wheels.yml

```diff
@@ -1,4 +1,4 @@
-name: Build
+name: "Build and publish Python wheels"
```

Comment:
Not a problem, but quoting strings is only needed in yml if they include special characters such as :

Reply:
Good to know!